src/commonTest/kotlin/exchange.dydx.abacus/app/helper/TriggerOrderToastGeneratorTests.kt

```diff
@@ -157,6 +157,7 @@ class TriggerOrderToastGeneratorTests {
                         price = 1000.0,
                         triggerPrice = 1500.0,
                         size = 1.0,
+                        sizeInput = null,
```

Comment:
can we add a sizeInput for one of these values to confirm that it's piped through properly

Reply:
oh this should always be null, because trigger orders always have a null [sizeInput](https://github.com/dydxprotocol/v4-abacus/pull/530/files#diff-f684fa19a17638d5b6d17dfd61dd715ab96db87920846ac4c92790e9cd0b8506R1183)

Reply:
thank you! if it ends up being an enormous lift i wouldn't worry about it, but it'd probably be worth tagging somewhere in a comment that we want to test this to make sure we dont have regressions later